airbyte-integrations/java-template-destination/readme.md

```diff
@@ -0,0 +1,20 @@
+### How to write a destination integration in java.
```

Comment:
can we have a script for that?

Reply:
(could be a good first issue)

Reply:
yes. we can definitely write a script to do this.

Reply:
but not going to do that in this PR.

Reply:
Yep, let's make it an issue and tagging it as "good first issue" (or smth like that)

Reply:
issue for this: https://github.com/airbytehq/airbyte/issues/512